HW_3/main.py

```diff
@@ -0,0 +1,36 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask import Flask, render_template, request, redirect, url_for
+from flask_wtf import CSRFProtect
+from forms import RegistrationForm
+from models import db, User
+from hashlib import sha256
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///mydatabase.db'
+app.config['SECRET_KEY'] = b'fefuhf7382f8y63tfgy682tfg3yfoa6gt3of6tr35frt7t2gf763tgf6tg'
+
+csrf = CSRFProtect(app)
+
+db.init_app(app)
+
+@app.cli.command('init-db')
+def db_init():
+    db.create_all()
+
+@app.route('/')
+def index():
+    return render_template('base.html')
+
+@app.route('/register/', methods=['GET', 'POST'])
+def register():
+    form = RegistrationForm()
+    if request.method == 'POST' and form.validate():
```

Comment:
Валидацию можно сделать так
```suggestion
    if form.validate_on_submit()
```